src/components/section/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react';
+import {
+  Box,
+  BoxProps,
+  chakra,
+  RecipeVariantProps,
+  useSlotRecipe,
+} from '@chakra-ui/react';
+
+import { sectionRecipe } from 'src/themes/shared/slotRecipes/section';
+
+import { Stack } from '../stack';
+
+type SectionVariantProps = RecipeVariantProps<typeof sectionRecipe>;
+
+export type SectionProps = SectionVariantProps & {
+  variant?: 'hero' | 'regular';
```

Comment:
You're already extending recipe props which includes `variant`, so this is not needed anymore
```suggestion
```